server/src/modules/session/util.service.ts

```diff
@@ -44,6 +45,21 @@ export class SessionUtilService {
     protected readonly encryptionService: EncryptionService,
     protected readonly jwtService: JwtService
   ) {}
+
+  async setAuditLogForUser(userId: string, manager: EntityManager): Promise<void> {
```

Comment:
If there are no additional session-related audit log events, you can call it directly from the service function. Alternatively, if the function in users.util.server.ts shares the same logic, please use a common utility function across multiple services.

Reply:
Currently the function shares the same logic for the most part because of default properties. But we plan on having additional properties for all the actions so sharing the same function may seem a bit difficult.
I'll try to reuse the function wherever possible.